spec/other/major_deprecation_spec.rb

```diff
@@ -178,17 +178,33 @@
 
     context "with github gems" do
       it "warns about the https change" do
-        msg = "The :github option uses the git: protocol, which is not secure. " \
-        "Bundler 2.0 will use the https: protocol, which is secure. Enable this change now by " \
-        "running `bundle config github.https true`."
+        msg = <<-EOS
+The :github git source is deprecated, and will be removed in Bundler 2.0. Change any "reponame" :github sources to "username/reponame". Add this code to your Gemfile to ensure it continues to work:
```

Comment:
I think we need "Add this code at the beginning of your Gemfile", because the git source has to be declared before it is used.